lib/metasploit/framework/login_scanner/zabbix.rb

```diff
@@ -49,7 +49,7 @@ def check_setup
             end
 
             if res.body.to_s !~ /Zabbix ([^\s]+) Copyright .* by Zabbix/m # Regex check for older versions of Zabbix prior to version 3.
-              if res.body.to_s !~ /<a target="_blank" class="grey link-alt" href="http[sS]{0,1}:\/\/www\.zabbix\.com\/documentation\/(\d+\.\d+)\/">Help<\/a>/m
+              if res.body.to_s !~ /href="http[sS]{0,1}:\/\/www\.zabbix\.com\/documentation\/(\d+\.\d+)\/">Help<\/a>/m
```

Comment:
To a future traveller, extra details are on the raised issue - https://github.com/rapid7/metasploit-framework/issues/17218

> What's on the page in version 6.2.4:
> <a target="_blank" rel="noopener noreferrer" class="grey link-alt" href="https://www.zabbix.com/documentation/6.2/">Help</a>
>
> Note the rel attribute has been added to the Help hyperlink tag: rel="noopener noreferrer"